docker-compose.yml

```diff
@@ -82,6 +78,7 @@ services:
     ports:
       - "5432:5432"
     environment:
+      POSTGRES_HOST: postgres
```

Comment:
Is this actually used anywhere?

Reply:
Not anymore, good spotting :) I remove it, alongside with other unused env variables :)